contracts/contracts/gateway/GatewayManagerFacet.sol

```diff
@@ -27,6 +27,8 @@ contract GatewayManagerFacet is GatewayActorModifiers, ReentrancyGuard {
     using AssetHelper for Asset;
     using EnumerableSet for EnumerableSet.Bytes32Set;
 
+    event SubnetKilled(SubnetID id);
```

Comment:
Let's name this event `SubnetDestroyed`. That term is more general as it can apply in ephemeral subnets, self-destruction, expirations (e.g. subnet is not bootstrapped in a predefined amount of time).